src/components/TabAbout/Contact/index.jsx

```diff
@@ -140,20 +142,36 @@ function Contact() {
                                         <i className="feather-phone me-2"></i>
                                         {userInfo.phoneNumber}
                                     </h4>
-                                    <i onClick={() => setPhoneStatus(!phoneStatus)}
-                                       className="ti-pencil d-flex font-md float-right cursor-pointer hover-edit"></i>
+                                    {userInfo.id === currentUser.id ?
+                                        <i onClick={() => setPhoneStatus(!phoneStatus)}
+                                           className="ti-pencil d-flex font-md float-right cursor-pointer hover-edit">
+                                        </i>
+                                        : null
```

Comment:
Should return fragment tag <></> instead of null value

Reply:
Resolved!